prow/git/v2/interactor.go

```diff
@@ -236,6 +236,16 @@ func (i *interactor) mergeMerge(commitlike string, opts ...MergeOpt) (bool, erro
 	return false, nil
 }
 
+func (i *interactor) mergeMerge(commitlike string, opts ...MergeOpt) (bool, error) {
```

Comment:
How about naming this as "mergeNoFF" or "mergeAlways"?

Reply:
I also thought the name mergemerge was weird lol, but the naming convention for these seem to be Merge(MergeType). I don't think it is necessary to stick to this, but I'd rather not change the name if we don't need to.

Reply:
> the naming convention for these seem to be Merge(MergeType)

Right, which is why I think `mergeNoFF` or `mergeNoFastForward` is a better name than `mergeMerge` because `Merge` is not a merge type AFAICS.

But also, `mergeIfNecessary` is not a good name because it has an if-condition built into its name. A better name is `mergeFF` or `mergeFastForward`.

Reply:
I agree that typically mergeIfNecessary would not be a great name, but since the name of the strategy gerrit uses is also called mergeIfNecessary. I'd rather leave it like this so that people understand that this method is simulating the mergeIfNecessary strategy rather than having to know that mergeFF == mergeIfNecessary

Reply:
> name of the strategy gerrit uses is also called mergeIfNecessary

Ah, that certainly changes things. SGTM, thanks!